tools/speech_dataset_processor/sdp/processors/modify_manifest/modify_manifest.py

```diff
@@ -54,10 +62,10 @@ def test(self):
                 )
 
     @abstractmethod
-    def _process_dataset_entry(self, data_entry):
+    def _process_dataset_entry(self, data_entry: Type[DataEntry]):
```

Comment:
As mentioned above, this is not really of type DataEntry (most often it's a simple dict, but can be anything in principle)